jvm-packages/xgboost4j-gpu/src/test/resources/iris.data.csv

```diff
@@ -0,0 +1,150 @@
+5.1,3.5,1.4,0.2,0.0
```

Comment:
Please fetch any test data you need instead of committing them into git history.